src/Microsoft.Data.Sqlite/SqliteEngine.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Data.Sqlite.Interop;
+
+namespace Microsoft.Data.Sqlite
+{
+    public static class SqliteEngine
```

Comment:
> No, it's not per-connection

...currently. But can't we dynamically switch to a different `ISqlite3` implementation?

Connection string seems like natural place to put this kind of configuration. Ideally the setting (something like`UseWinsqlite3=True;`) could handle the "if windows 10 do this, else do that"  for users so we wouldn't need different connection strings for Linux, Windows 7 and Windows 10.